author.php

```diff
@@ -10,75 +10,52 @@
  * @since 1.2.1
  */
 
-get_header(); ?>
-
-	<section id="primary" class="site-content">
-		<div id="content" role="main">
-
-		<?php if ( have_posts() ) : ?>
-
-			<?php
-
-				/*
-				 * Queue the first post, that way we know
-				 * what author we're dealing with (if that is the case).
-				 *
-				 * We reset this later so we can run the loop
-				 * properly with a call to rewind_posts().
-				 */
-				the_post();
-			?>
-
-			<header class="archive-header">
-				<h1 class="archive-title">
-					Author Archives:
-					<span class="vcard">
-						<a class="url fn n" href="<?php echo esc_url( get_author_posts_url( get_the_author_meta( 'ID' ) ) ); ?>" title="<?php echo esc_attr( get_the_author() ); ?>" rel="me">
+get_header();
+
+$sidebar_class = '';
+if ( is_active_sidebar( 'sidebar-1' ) ) {
+	$sidebar_class = 'has-sidebar';
+}
+?>
+
+<div id="stage" class="inner" role="main">
+	<div id="content" class="content <?php echo esc_attr( $sidebar_class ); ?>">
+
+		<div class="content-main main-content">
+
+			<?php if ( have_posts() ) : ?>
+				<?php
+
+					/*
+					 * Queue the first post, that way we know
+					 * what author we're dealing with (if that is the case).
+					 *
+					 * We reset this later so we can run the loop
+					 * properly with a call to rewind_posts().
+					 */
+					the_post();
+				?>
+				<header class="archive-header">
+					<h1 class="archive-title">
+						Author Archives:
+						<span class="vcard">
 							<?php the_author(); ?>
-						</a>
-					</span>
-				</h1>
-			</header><!-- .archive-header -->
+						</span>
+					</h1>
+				</header><!-- .archive-header -->
+			<?php endif; ?>
 
 			<?php
-
-				/*
-				 * Since we called the_post() above, we need to
-				 * rewind the loop back to the beginning that way
-				 * we can run the loop properly, in full.
-				 */
-				rewind_posts();
+			/* We removed the loop because we don't use this display template. */
```

Comment:
That's very helpful context for this review, but I'm not sure it needs to remain in the codebase.